index.html

```diff
@@ -121,6 +130,115 @@ <h2 class="txt-center">
     $('.js-defaultServerIndexApp').on('click', function(){
       app.loadNewServer("http://localhost:18469/api/v1/");
     });
+
+    var appMenu = Menu.buildFromTemplate([
```

Comment:
I wasn't able to get it to recognize the right click on the BrowserWindow in the main main.js. I can take another look at it.